frontend/apps/reader/modules/readerfont.lua

```diff
@@ -210,12 +210,14 @@ function ReaderFont:onSetFontSize(new_size)
     if new_size > 255 then new_size = 255 end
     if new_size < 12 then new_size = 12 end
 
-    self.font_size = new_size
-    self.ui.document:setFontSize(Screen:scaleBySize(new_size))
-    self.ui:handleEvent(Event:new("UpdatePos"))
-    UIManager:show(Notification:new{
-        text = T( _("Font size set to %1."), self.font_size),
-    })
+    if self.ui.document then
```

Comment:
If you managed to get that happening, we have a problem with Dispatcher :)
And if it may happen that Dispatcher sends "reader" events, better check for ui.document in Dispatcher before sending "reader" events (as it must know it's a reader event with its padding=true, rolling=true).

If that wouldn't be possible, all event handlers (that would access self.ui.document) would be candidate for crashes.
If this would need to be fixed in each of them, better just add a single line at their top:
`if not self.ui.document then return end`
so to not cause indendation like here and mess with git blame.